corehq/form_processor/tests/test_migration.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+from django.core.management import call_command
+from django.test import TestCase
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors, CaseAccessors
+from corehq.form_processor.tests.utils import FormProcessorTestUtils
+from corehq.form_processor.utils import should_use_sql_backend
+from corehq.util.test_utils import create_and_save_a_form, create_and_save_a_case
+
+
+class MigrationTestCase(TestCase):
+
+    def setUp(self):
+        FormProcessorTestUtils.delete_all_cases_forms_ledgers()
+        self.domain = uuid.uuid4().hex
+
+    def test_basic_form_migration(self):
+        create_and_save_a_form(self.domain)
+        self.assertFalse(should_use_sql_backend(self.domain))
+        self.assertEqual(1, len(FormAccessors(domain=self.domain).get_all_form_ids_in_domain()))
+        self._do_migration_and_assert_flags(self.domain)
+        self.assertEqual(1, len(FormAccessors(domain=self.domain).get_all_form_ids_in_domain()))
+        # todo: verify form properties?
```

Comment:
seems like we should have at least one test do a diff of the form json just for sanity